assignment1/src/scripts/letter.ts

```diff
@@ -0,0 +1,27 @@
+//Class that represents a letter in the bootleg wordle game
+export enum LetterStatus {
+    Unknown = 0,
+    Correct,
+    Misplaced,
+    Wrong
+}
+export class letter {
+    letter: string;
+    status: LetterStatus = LetterStatus.Unknown;
+
+    constructor(letter: string){
+        this.letter = letter;
+    }
+
+    get color(){
+        switch(this.status){
+            case LetterStatus.Correct:
+                return 'green';
+            case LetterStatus.Misplaced:
+                return 'gold';
+            case LetterStatus.Wrong:
+                return 'black';
```

Comment:
This might be a stylistic choice but we have been using orange for misplaced and red for wrong in class per the actual game.

Reply:
Orange vs gold is mostly stylistic and not too big of a deal in my opinion. While html's default "orange" is basically a golden color, in my mind I envision default orange colors as like highlighter/safety orange or pumpkin orange, neither of which are accurate to the color we're aiming to recreate. And from what I saw the game doesn't mark incorrect letters as red it simply darkens them/greys them out (meaning where the default is already grey it must be "darker grey" which is easier to default to "black" when using one word basic colors not like "smokey grey" or something). Although red does convey the same sense of "hey this is incorrect" so I think either would work honestly.

As they didn't specify for this assignment anything to do with the letters/colors, I'm going to leave it because this is how it makes sense to me. But in the coming assignments if necessary I will change it to fit their example.